frontend/src/util/localDocStore.js

```diff
@@ -23,6 +30,7 @@ const updateDocumentBody = (docId, state) => {
 
   const docHashes = JSON.parse(docHashesString);
   docHashes[docId].state = state;
+  docHashes[docId].docState = DOC_STATE.InProgress;
```

Comment:
only if it's not completed? unless we want edits on completed docs to put them back into in progress

Reply:
I mean, that's what how I thought about it. Like, sure :)